packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-line.tsx

```diff
@@ -130,7 +130,8 @@ type MovableLineProps = {
           };
 };
 
-export const MovableLine = (props: MovableLineProps) => {
+// Exported for testing
+export const Line = (props: LineProps) => {
```

Comment:
I don't love the way this code is split up - it's hard to find a name for this internal component that makes sense and clarifies how its purpose is different from that of `MovableLine`. But this `Line` component has tests pinning it down, so refactoring across the two components is costly.